.github/workflows/test.yml

```diff
@@ -32,3 +39,39 @@ jobs:
         env_vars: PLATFORM,PYTHON
         name: codecov-umbrella
         fail_ci_if_error: false
+
+  pypi:
+    needs: [test]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
```

Comment:
I think we should test on all versions the package supports, ideally 3.6 to 3.9 to be aligned with Oríon.

Reply:
You commented on the python version that is [coming from Orion](https://github.com/Epistimio/orion/blob/develop/.github/workflows/build.yml#L159) :p

Reply:
we have 3.10 too so when orion drops 3.6 by 3.10 we will be ready